cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/AppFabricServiceMain.java

```diff
@@ -118,7 +119,8 @@ protected void configure() {
           bind(MetadataPublisher.class).to(MessagingMetadataPublisher.class);
           bind(MetadataServiceClient.class).to(DefaultMetadataServiceClient.class);
         }
-      }
+      },
+      new MetadataServiceModule()
```

Comment:
Why needed? What is this for? If it is just for using the client, it is better have a client module.

Reply:
Thanks, this is not required anymore, was from initially trying to directly use MetadataAdmin.

Reply:
Done